docs/bpmn-support-roadmap.adoc

```diff
@@ -30,7 +30,7 @@ BPMN elements
 * events
 ** [x] start event (0.1.0)
 ** [x] end event (0.1.1)
-** [ ] terminate end event
+** [x] terminate end event (0.1.2)
```

Comment:
:information_source: I suggest to rebase this PR on master as there is also another page to update related to the bpmn support

Reply:
done by me, see 28a74a2